styles/discuit.org/catppuccin.user.less

```
@@ -0,0 +1,197 @@
+==/UserStyle== */
+
+@-moz-document regexp("^https?://discuit\\.(org|net)/.*") {
+  html.theme-dark {
+    #catppuccin(@darkFlavor);
+  }
+  html:not(.theme-dark) {
+    #catppuccin(@lightFlavor);
+  }
+
+  #catppuccin(@flavor) {
+    @rosewater: @catppuccin[@@flavor][@rosewater];
+    @flamingo: @catppuccin[@@flavor][@flamingo];
+    @pink: @catppuccin[@@flavor][@pink];
+    @mauve: @catppuccin[@@flavor][@mauve];
+    @red: @catppuccin[@@flavor][@red];
+    @maroon: @catppuccin[@@flavor][@maroon];
+    @peach: @catppuccin[@@flavor][@peach];
+    @yellow: @catppuccin[@@flavor][@yellow];
+    @green: @catppuccin[@@flavor][@green];
+    @teal: @catppuccin[@@flavor][@teal];
+    @sky: @catppuccin[@@flavor][@sky];
+    @sapphire: @catppuccin[@@flavor][@sapphire];
+    @blue: @catppuccin[@@flavor][@blue];
+    @lavender: @catppuccin[@@flavor][@lavender];
+    @text: @catppuccin[@@flavor][@text];
+    @subtext1: @catppuccin[@@flavor][@subtext1];
+    @subtext0: @catppuccin[@@flavor][@subtext0];
+    @overlay2: @catppuccin[@@flavor][@overlay2];
+    @overlay1: @catppuccin[@@flavor][@overlay1];
+    @overlay0: @catppuccin[@@flavor][@overlay0];
+    @surface2: @catppuccin[@@flavor][@surface2];
+    @surface1: @catppuccin[@@flavor][@surface1];
+    @surface0: @catppuccin[@@flavor][@surface0];
+    @base: @catppuccin[@@flavor][@base];
+    @mantle: @catppuccin[@@flavor][@mantle];
+    @crust: @catppuccin[@@flavor][@crust];
+    @accent: @catppuccin[@@flavor][@@accentColor];
+
+    --base-bg: #rgbify(@base)[];
+    --base-fg: #rgbify(@text)[];
+    --base-brand: #rgbify(@accent)[];
+    --base-red: #rgbify(@red)[];
+    --base-green: #rgbify(@green)[];
+    --base-gray: #rgbify(@overlay1)[];
+
+    --color-bg: @mantle;
+    --color-fg: @text;
+    --color-brand: @accent;
+
+    --color-red: @red;
+    --color-green: @green;
+    --color-blue: @blue;
+
+    --color-card: @surface0;
+    --color-card-border: @surface0;
+    --color-card-domain: @red;
+
+    --color-input: fade(@overlay0, 15%);
+    --color-input-active: fade(@overlay0, 10%);
+    --color-link: @blue;
+
+    --color-button: fade(@overlay1, 70%);
+    --color-button-text: @text;
+    --color-button-main: @accent;
+    --color-button-main-hover: fade(@accent, 80%);
+    --color-button-red: @red;
+
+    --color-supporter: @pink;
+    --color-skeleton: fade(@overlay0, 5%);
+
+    --color-body: @base;
+    --color-text: @text;
+    --color-text-light: @subtext0;
+
+    --color-input-border: fade(@overlay0, 10%);
+    --color-input-border-hover: fade(@overlay0, 20%);
+    --color-input-border-focus: @accent;
+    --color-mini-footer-text: @subtext1;
+
+    color-scheme: if(@flavor = latte, light, dark);
+
+    ::selection {
+      background-color: fade(@overlay2, 30%);
```

```suggestion
      background-color: fade(@accent, 30%);
```

I'm not sure if you changed this intentionally or not but in the template we use accent as the selection color.